### ISSUE TITLE
Arm64 Native Builds For Windows

### ISSUE THREAD

**Issue by nmnmcc** [mifi/lossless-cut#1475]
### I have a lot of issues to go through, so in order to make it easier for me to help you, I ask that you please try these things first

- [X] Check out all the menus and keyboard shortcuts. Maybe the feature already exists?
- [X] Try with the [newest version from GitHub](https://github.com/mifi/lossless-cut/releases/latest)
- [X] Read the [documentation](https://github.com/mifi/lossless-cut) and [FAQ, Known issues, Troubleshooting](https://github.com/mifi/lossless-cut/blob/master/issues.md) to see if the feature already exists
- [X] See if someone already suggested this under [Issues](https://github.com/mifi/lossless-cut/issues) or [Discussions](https://github.com/mifi/lossless-cut/discussions)

### Description

I am currently using a Windows PC with a Qualcomm Snapdragon processor.
I would like to be able to get a native build for arm64 in the release.
Thanks!

**Comment by mifi** [comment]
**maybe** it would just work if enabling arm64 for windows in electron-builder under package.json:
https://github.com/mifi/lossless-cut/blob/dd1b44f2d3d645056df78175b421aff414b103e2/package.json#L277-L280

similar to linux: https://github.com/mifi/lossless-cut/blob/dd1b44f2d3d645056df78175b421aff414b103e2/package.json#L409-L429

but i'm not sure how to get ffmpeg arm64 windows builds. maybe it would have to be a bring-your-own-ffmpeg, like we already do for linux arm.

https://www.electronjs.org/docs/latest/tutorial/windows-arm

**Comment by yutouyes** [comment]
> **maybe** it would just work if enabling arm64 for windows in electron-builder under package.json:
> 
> https://github.com/mifi/lossless-cut/blob/dd1b44f2d3d645056df78175b421aff414b103e2/package.json#L277-L280
> 
> similar to linux:
> 
> https://github.com/mifi/lossless-cut/blob/dd1b44f2d3d645056df78175b421aff414b103e2/package.json#L409-L429
> 
> but i'm not sure how to get ffmpeg arm64 windows builds. maybe it would have to be a bring-your-own-ffmpeg, like we already do for linux arm.
> 
> https://www.electronjs.org/docs/latest/tutorial/windows-arm

Hello is there any update? I found some arm64 ffmpeg libs. Hope they can help.
https://github.com/System233/ffmpeg-msvc-prebuilt/releases
https://github.com/tordona/ffmpeg-win-arm64/releases

**Comment by mifi** [comment]
this hasn't been a very frequently requested feature (and there are no thumbs up on this issue), so I'm not sure of the value. If only a few people want it, it should still be possible to build it yourself by changing the code like i [described here](https://github.com/mifi/lossless-cut/issues/1475#issuecomment-1434383290) and then build it locally:

```bash
cd lossless-cut
yarn install
yarn build && npx electron-builder --win zip
```

**Comment by jullimullins** [comment]
Windows arm version are getting more popular, so I think that the need is increasing all the time. It also sound like there is a working solution available (This is just a hupotesis made by newbie).

**Comment by mifi** [comment]
still only 2 👍 on the op though

**Comment by jullimullins** [comment]
That's true. Question is that how many just use some other option instead of puting a request in ant it's also possible to run losslesscut xin64 version on arm computer through emulation. Emulation is actually working quite well.

**Comment by mifi** [comment]
This is avaliable in the latest nightly builds, see  #2788
